docs/sql/reference/kill.txt

```diff
@@ -37,6 +36,15 @@ killed because the ``KILL`` command is processed before the final result is
 sent to the client.
 
 
-``KILL ALL`` returns the number of jobs killed per node. For example if the
```

Comment:
``KILL ALL`` and ``KILL job_id`` return the number of **contexts** killed per node.